invenio_cli/commands/translations.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Cli is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio module to ease the creation and management of applications."""
+
+from pathlib import Path
+
+from ..commands import Commands
+from .steps import CommandStep
+
+
+class TranslationsCommands(Commands):
+    """Translations CLI commands."""
+
+    CMD_PREFIX = ["pipenv", "run"]
+
+    def extract(
+        self,
+        babel_file=None,
+        msgid_bugs_address=None,
+        copyright_holder=None,
+        add_comments=None,
+    ):
+        """Extract messages from source code and templates."""
+
+        msgid_bugs_address = msgid_bugs_address or self.cli_config.get_author_email()
```

Comment:
Should the commands receive just parameters? and have no knowledge about the `cli_config`? We are passing it across the whole library but feels wrong...